swagger.yml

```diff
@@ -1076,6 +1076,22 @@ paths:
             $ref: '#/definitions/deploy'
         default:
           $ref: '#/responses/error'
+  /deploys/{deploy_id}/ready:
+    post:
+      operationId: readyDeploy
+      tags: [deploy]
+      parameters:
+        - name: deploy_id
+          type: string
+          in: path
+          required: true
+      responses:
+        '201':
```

Comment:
Not sure why on [the BB side](https://github.com/netlify/bitballoon/pull/16569/files#diff-c6ed00a289567dfc399f1032e78413680734d0cb65230b37df98d72e615f4663R368) we are saying a 200 for this and the rest of operations that update the deploy state

Reply:
I don't quite follow. 200 seems the correct response in most cases. Could be a copy-pasta error in others? Seems like we should respect 200 here, right?

Reply:
It seems to be the case, updated ready, cancel and restore endpoint to match what we really return, as 201 should be when a resource it's created.